styleguide.config.js

```diff
@@ -81,6 +81,10 @@ module.exports = {
         {
           name: 'Forms',
           components: 'src/organisms/forms/**/*.js'
+        },
+        {
+          name: 'Tables',
+          components: () => [ 'src/organisms/tables/ComparisonTable/index.js' ]
```

Comment:
this was to prevent the styleguide from rendering `ComparisonTableRow`, but I changed the blob to only pull index.js so I think this will solve the problem and we should be good going forward